### DISCUSSION TITLE
Question - Seq to seq

### DISCUSSION THREAD

**Discussion by davodogster** [timeseriesAI/tsai#316] [Q&A]
Hi, awesome repo! 
Does the repo allow for sequence to sequence models (also known as sequence segmentation)?

**Comment by oguiza** [comment]
> Hi, awesome repo!

Thanks!

> Does the repo allow for sequence to sequence models (also known as sequence segmentation)?

If you are looking for an encoder-decoder type of architecture the answer is no. We don't still have any. 
What we have are architectures (all ending in Plus) that allow you to pass a target of fixed length, and train the model to generate those targets. Targets can have 0, 1 or 2 dimensions (single point, vector or matrix).
I don't know if this answers your question.

**Comment by oguiza** [comment]
I'll close this issue due to a lack of response. Please, reopen if necessary.

**Comment by davodogster** [comment]
@oguiza Hi, here is one example of what I mean https://www.mathworks.com/help/signal/ug/waveform-segmentation-using-deep-learning.html seq2seq , time series segmentation. Cheers

**Comment by vrodriguezf** [comment]
Have a look at this [discussion thread](https://github.com/timeseriesAI/tsai/discussions/61)
